### ISSUE TITLE
Unable to buyout auction with previous bid

### ISSUE THREAD

**Issue by cjwyatt** [dalaranwow/dalaran-wow#5277]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
When trying to buyout an auction for which I am currently the highest bidder, the buyout button lights up and I can click it and be prompted to buy the item. After clicking Accept nothing happens.
**Expected behaviour**: Tell us what should happen instead
My character should expend the difference between my current highest bid and the buyout price extra to buyout the auction and I should then win the auction.
**Steps to reproduce the problem**:

1. Place a bid on an item
2. Attempt to buyout the item (I have less extra gold than the full buyout price)

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

![image](https://user-images.githubusercontent.com/6775797/35484550-5c42d36a-0449-11e8-94d9-6e394d70da97.png)

![image](https://user-images.githubusercontent.com/6775797/35484558-6b6a3af4-0449-11e8-8ac8-91a72a1a64c8.png)

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by AlexCSCP** [comment]
> My character should expend the difference between my current highest bid and the buyout price extra to buyout the auction and I should then win the auction.
Just wanna add that is does work as expected when you have more gold than the buyout price.
